src/pool/srv_pool.c

```diff
@@ -1070,7 +1070,7 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf,
 	struct rdb_tx	tx;
 	d_iov_t		value;
 	bool		version_exists = false;
-	uint32_t	version;
+	uint32_t	version, global_version;
```

Comment:
[Nit; just FYI] Should be added on a separate line

     uint32_t        version;
    +uint32_t        global_version;